### ISSUE TITLE
Feature request (Assigning tickets)

### ISSUE THREAD

**Issue by scsaus** [opensupports/opensupports#80]
Hi it will be great if admin could assign tickets to other staff members from admin panel and also can see all tickets if though tickets were assigned to other staff member.Can you add this feature ?

**Comment by scsaus** [comment]
Any one ?

**Comment by ivandiazwm** [comment]
Yes, this is an feature we were thinking about. It would be added to our feature kanban for a future release
